### PR TITLE
Enhance CKEditor with more options (task #1677)

### PR DESCRIPTION
**Pull request by flangofas**

### CODE REVIEW

**Comment by georgeconstantinou** [comment]
:+1:

File: src/Template/Element/ckeditor.ctp [hunk 1]
```diff
@@ -1,33 +1,22 @@
...
-            // Configure the Enhanced Image plugin to use classes instead of styles and to disable the
-            // resizer (because image size is controlled by widget styles or the image takes maximum
-            // 100% of the editor width).
-            image2_alignClasses: ['text-left', 'text-center', 'text-right'],
-            image2_disableResizer: true
-        };
-EOT;
-}
-
+echo $this->Html->script('//cdn.ckeditor.com/4.5.8/full-all/ckeditor.js', ['block' => 'scriptBottom']);
```
Comment (mamchenkov) [inline src/Template/Element/ckeditor.ctp]: @flangofas please use HTTPS ... reasons:

1. Not all browsers support protocol-less URLs. (though the major ones do)
2. Easier to grep for URLs in the code (`grep -ir http .`)
3. Single cached entry in the browser (not one for HTTP and one for HTTPS)
4. Everything we do will be on HTTPS [Real Soon Now](https://letsencrypt.org/) anyway  :)

File: webroot/js/ckeditor-upload-plugin.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+// When there is an upload URL then we can enable
+// CKeditor's Image Paste plugin which allows to upload pasted or dropped images
+// @link: http://ckeditor.com/addon/imagepaste
+var uploadOptions = {
+    extraPlugins: 'uploadimage,image2',
+    // Load the default contents.css file plus customizations for this sample.
+    contentsCss: [ CKEDITOR.basePath + 'contents.css', 'http://sdk.ckeditor.com/samples/assets/css/widgetstyles.css' ],
```
Comment (mamchenkov) [inline webroot/js/ckeditor-upload-plugin.js]: @flangofas this will break on projects which are on HTTPS, right?

Comment (flangofas) [inline webroot/js/ckeditor-upload-plugin.js]: @mamchenkov yeap. It is actually unnecessary styles which i didn't notice :+1:  thanks

**Pull request merged**